proxy-config/configure_onion_service.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
Would be nice to have an integration test for this, if we move this into proxy/ then we can use the existing pytest stuff.